macros/models/_sector/transfers/eth/transfers_eth.sql

```diff
@@ -0,0 +1,71 @@
+{% macro transfers_eth(blockchain, eth_placeholder_contract, base_traces, base_transactions, erc20_transfer ) %}
```

Comment:
erc20_transfer should maybe have a true/false flag on whether to look at ti?

Legacy OVM1 should be the only chain listed here that had ETH as an erc20 at some point, but other L2s might in the future.

I don't want to accidently pull in weird ERC20 things for other chains.

Reply:
Do you mean this part only applies to OVM one but not the rest of the OP chains?

```
--ETH Transfers from deposits and withdrawals are ERC20 transfers of the 'deadeadead' ETH token. These do not appear in traces.
```

Reply:
I'm intending to close this PR of creating macro or maybe just modify the originals for some cleanup. Think the combined `tokens` transfer that the Dune team is been working on will have both ETH transfers and erc20s combined cross chain onwards